drivers/SmartThings/philips-hue/src/hue/api.lua

```diff
@@ -105,7 +106,9 @@ function PhilipsHueApi.new_bridge_manager(base_url, api_key, socket_builder)
     while true do
       local msg, err = control_rx:receive()
       if err then
-        log.error_with({ hub_logs = true }, "[PhilipsHueApi] Error receiving on control channel for REST API thread", err)
+        if err ~= "timeout" then
```

Comment:
Do we still want to log the timeout error for visibility?

Reply:
For this channel we expect timeouts as part of normal operation; that would just mean that a command hasn't been sent in a while. I'm not sure this specific timeout log would tell us anything, the other timeouts should be logged I believe though.